Tests/Tests/Encryption/Keychain/CDTEncryptionKeychainUtilsAESTests.m

```diff
@@ -0,0 +1,181 @@
+//
+//  CDTEncryptionKeychainUtilsAESTests.m
+//  Tests
+//
+//  Created by Enrique de la Torre Fernandez on 10/04/2015.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file
+//  except in compliance with the License. You may obtain a copy of the License at
+//  http://www.apache.org/licenses/LICENSE-2.0
+//  Unless required by applicable law or agreed to in writing, software distributed under the
+//  License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+//  either express or implied. See the License for the specific language governing permissions
+//  and limitations under the License.
+//
+
+#import <XCTest/XCTest.h>
+
+#import "CDTEncryptionKeychainUtils+AES.h"
+
+#import "TDBase64.h"
+
+@interface CDTEncryptionKeychainUtilsAESTests : XCTestCase
+
+@property (strong, nonatomic) NSData *key;
+@property (strong, nonatomic) NSData *iv;
+
+@end
+
+@implementation CDTEncryptionKeychainUtilsAESTests
+
+- (void)setUp
+{
+    [super setUp];
+
+    // Put setup code here. This method is called before the invocation of each test method in the
+    // class.
+    NSString *keyStr = @"3271b0b2ae09cf10128893abba0871b64ea933253378d0c65bcbe05befe636c3";
+    NSString *ivStr = @"10327cc29f13539f8ce5378318f46137";
+
+    self.key = [CDTEncryptionKeychainUtilsAESTests dataFromHexadecimalString:keyStr];
+    self.iv = [CDTEncryptionKeychainUtilsAESTests dataFromHexadecimalString:ivStr];
+}
+
+- (void)tearDown
+{
+    // Put teardown code here. This method is called after the invocation of each test method in the
+    // class.
+    self.key = nil;
+    self.iv = nil;
+
+    [super tearDown];
+}
+
+- (void)testDoEncrypt
+{
+    NSString *txt = @"1234567890";
+    NSString *expectedResult = @"ExUL75zXTBiQKHAqYeR3Glt+EXMR25qmNTdeToHdA40=";
```

Comment:
Name with more meaningful names:

- `txt` -> `plainText`
- `expectedResult` -> `expectedCiphertext`

Helps because below you reuse the name `txt` for the input ciphertext rather than the plain text.